files/scripts/swss.sh

```diff
@@ -292,6 +292,7 @@ start() {
     # Don't flush DB during warm boot
     if [[ x"$WARM_BOOT" != x"true" ]]; then
         debug "Flushing APP, ASIC, COUNTER, CONFIG, and partial STATE databases ..."
+        clean_up_tables APPL_DB "'PORT_TABLE:E*'"
```

Comment:
@mihirpat1 is there a difference in behavior when swss start as part of cold reboot vs as part of swss/OA crash?

Reply:
@prgeor - No, I don't see a difference in behavior between between cold reboot v/s SWSS/OA crash

Reply:
@mihirpat1 can you add the regex for `EthernetXX`

Reply:
I have addressed this now.